core/execution/future/market.go

```diff
@@ -3364,10 +3370,54 @@ func (m *Market) submitStopOrders(
 ) []*types.OrderConfirmation {
 	confirmations := []*types.OrderConfirmation{}
 	evts := make([]events.Event, 0, len(stopOrders))
+	toDelete := []*types.Order{}
 
-	// might contains both the triggered orders and the expired OCO
+	// might contain both the triggered orders and the expired OCO
 	for _, v := range stopOrders {
 		if v.Status == status {
+			// If we have an order size override, handle that here
+			if v.SizeOverrideSetting == types.StopOrderSizeOverrideSettingOrder {
+				// Update the order size to match that of the referenced order
+				order, _, err := m.getOrderByID(v.SizeOverrideValue.OrderID)
+				if err != nil {
+					m.log.Error("could not get order to override size with",
+						logging.StopOrderSubmission(v),
+						logging.Error(err))
+				}
+				v.OrderSubmission.Size = order.Size - order.TrueRemaining()
+
+				// Need to cancel the referenced order
+				toDelete = append(toDelete, order)
```

Comment:
We ignore the parameter returned which says if the order was in the book or not, however I don't think we should be submitting the order if it's not traded right? Also by that point, if the order has traded fully and available neither from the book or the pegged list, this will return nothing, we need for every order which is linked to a stop order to keep track of the actual volume which has been traded, and execute against this instead.

Reply:
I think if the referenced order is filled then the triggered order should do nothing as the purpose of it was to unwind a non complete order. But I'd like @TomMcL to answer this one fully.

Reply:
Linked orders no longer supported, closing.